### PR TITLE
Quiz app

### PR DESCRIPTION
**Pull request by murph1234**
## Related Issue
#114 
I wanted to make a few changes in the css as the page is not looking attractive and appealing as it should be with respect to a quiz game. I would like to make the texts on the page a bit more organized and also make some changes on the background image.
 

### Describe Changes
As I have described above I have made changes with the website layout by adding Bootstrap. This enhanced the font style and the overall look of the website. I have also made the buttons rounded by changing its border radius. I have completely modified the CSS of chemistry, computer and maths page by maintaing the texts as it should be( in center).Moreover when the result was showing a thick border of red , yellow and green was coming which was looking a bit odd.I changed it and rather made the text show the respective colors which is looking better.This change I made in the JS file of the respective subjects.

##Screenshots
image
 
The other pages are looking the same way!

### Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 I have starred the repo
- [x] 👀 I have read the [Contributing Guideline](https://github.com/Shweta2024/JavaScript-Projects/blob/master/ContributingGuidelines.md)

### What types of changes does your code introduce?
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by utkarsh006** [comment]
@murph1234 Tickmark checkboxes like this[x]

**Comment by murph1234** [comment]
@utkarsh006 done!!

**Comment by Shweta2024** [comment]
> @utkarsh006 done!!

@murph1234 
tick mark the checkboxes of "What types of changes does your code introduce?" like [x] this

**Comment by murph1234** [comment]
@Shweta2024 done!!

**Comment by utkarsh006** [comment]
@murph1234 Add README file.

**Comment by utkarsh006** [comment]
@murph1234 Answer my review above.

**Comment by murph1234** [comment]
@utkarsh006 I am not able to find the reviews

**Comment by utkarsh006** [comment]
## Here it is!
Screenshot (303)

**Comment by murph1234** [comment]
@utkarsh006 previously it was showing a block of color (red,yellow and green) respectively which was looking a bit odd with respect to the background image but now only the text is getting colored which is looking better

**Comment by utkarsh006** [comment]
@murph1234 Upload video of your project in README and add comments to the code.

File: Quiz-App/js/chemistry.js [hunk 1]
```diff
@@ -307,27 +307,27 @@ function res () {
     else incorrect++;
   }
   if (marks <= 4) {
-    document.getElementById("result").style.backgroundColor = "red";
+    document.getElementById("result").style.color = "red";
```
Comment (murph1234) [inline Quiz-App/js/chemistry.js:310]: previously it was showing a block of color (red,yellow and green) respectively which was looking a bit odd with respect to the background image but now only the text is getting colored which is looking better

**Comment by murph1234** [comment]
@utkarsh006 done!!

**Comment by utkarsh006** [comment]
@murph1234 Add comments to the JS code

**Comment by murph1234** [comment]
I had to do it in all the JS codes i have already done it in chemistry

**Comment by murph1234** [comment]
@utkarsh006

**Comment by utkarsh006** [comment]
@murph1234 I think rather than arguments you must follow guidelines. And write one message at a time for replying.

**Comment by murph1234** [comment]
@utkarsh006 done

**Comment by murph1234** [comment]
@utkarsh006 done

**Review by Shweta2024 — CHANGES_REQUESTED** [review]
resolve merge conflicts

**Comment by murph1234** [comment]
@Shweta2024 how to do it?

**Comment by utkarsh006** [comment]
@murph1234 Close the PR and raise another new PR following all the guidelines.

**Pull request closed without merging**